project-words.txt

```diff
@@ -0,0 +1,218 @@
+Agren
+Albornoz
+Altmetric
+apoc
+arrowless
+Avlant
+Banaei
+Benfeitas
+bigg
+Billa
+Bioconductor
+bioschemas
+boxe
+bulma
+Bulma
+ccms
+centering
+cerevisiae
+chebi
+CHEBI
+Cholley
+cmid
+cmids
+cmodel
+coelicolor
+comparemodels
+compm
+compo
+connectdevelop
+csvg
+csvgs
+cutrectangle
+Cvijovic
+cxttap
+cypan
+Cytoscape
+cyzoom
+datasource
```

Comment:
```suggestion
```
Do we really need this? You fixed some instances [in the code](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1094/commits/b5f5684dbab60cbe59bd374c90ba1eb6bd3fe39d#diff-d5a9a86af1198788249f566274014edcde65bdcb8575f405ba8c2b69471423ecL16) and I can't find any others. It would be nice to get warnings if we reintroduce the word.

Reply:
Fixed in 6e17ad9076beea0dd75248ac21f4c6636b9f58a8